build/ubuntu/Dockerfile.prebuilder

```diff
@@ -57,15 +57,17 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
         libncurses-dev \
         libncurses5 \
         libtinfo5 \
-        protobuf-compiler
+        protobuf-compiler \
+        rsync
 
 ARG GHC_VERSION=8.10.7
 RUN curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org \
     | BOOTSTRAP_HASKELL_NONINTERACTIVE=1 BOOTSTRAP_HASKELL_GHC_VERSION=${GHC_VERSION} sh
```

Comment:
Can we also pin the cabal version here? Maybe also expose a `BUILD_ARG` for that?